### PR TITLE
[TT-7505] update usage of mockResponse query parameter

### PR DESCRIPTION
**Pull request by jeffy-mathew**
Add usage of mockResponse query parameter to tyk-docs
https://tyktech.atlassian.net/browse/TT-7505

[TT-7505]: https://tyktech.atlassian.net/browse/TT-7505?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by lghiur — APPROVED** [review]

File: tyk-docs/content/getting-started/using-oas-definitions/import-an-oas-api.md [hunk 1]
```diff
@@ -646,6 +646,170 @@ For more information on how Tyk builds the middleware operations structure in or
...
 
+### Tutorial: Create an API with mock response enabled
+
+| Property     | Description            |
+|--------------|------------------------|
+| Resource URL | /tyk/apis/oas/import   |
+| Method       | POST                   |
+| Type         | None                   |
+| Body         | Tyk OAS API Definition |
+| Param        | mockResponse           |
```
Comment (letzya) [inline tyk-docs/content/getting-started/using-oas-definitions/import-an-oas-api.md]:
```suggestion
| Param        | `mockResponse`           |
```

File: tyk-docs/content/getting-started/using-oas-definitions/import-an-oas-api.md [hunk 2]
```diff
@@ -646,6 +646,170 @@ For more information on how Tyk builds the middleware operations structure in or
 
 You created an API which tells your Tyk Gateway to validate any incoming request against the JSON schema defined in the OAS API Definition.
 
+### Tutorial: Create an API with mock response enabled
+
+| Property     | Description            |
+|--------------|------------------------|
+| Resource URL | /tyk/apis/oas/import   |
```
Comment (letzya) [inline tyk-docs/content/getting-started/using-oas-definitions/import-an-oas-api.md]:
```suggestion
| Resource URL | `/tyk/apis/oas/import`   |
```

File: tyk-docs/content/getting-started/using-oas-definitions/import-an-oas-api.md [hunk 3]
```diff
@@ -646,6 +646,170 @@ For more information on how Tyk builds the middleware operations structure in or
...
+| Method       | POST                   |
+| Type         | None                   |
+| Body         | Tyk OAS API Definition |
+| Param        | mockResponse           |
+
+#### Import OAS API Definition and enable mockResponse
+
+You’re going to send an OAS API Definition to the Tyk Gateway import API, which also contains describes GET `/pet/{petId}` how the response would look like. In order for Tyk to read and mock any response against the JSON schema provided, `mockResponse=true` query parameter is needed when calling the import endpoint.
+
+```
```
Comment (letzya) [inline tyk-docs/content/getting-started/using-oas-definitions/import-an-oas-api.md]:
```suggestion
```.curl
```

File: tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md [hunk 4]
```diff
@@ -566,6 +566,173 @@ Go to the `/apps` folder of your Tyk Gateway installation (by default in `/var/t
...
+
+#### Mock response from OAS definition
+
+In the OAS API Definition that you updated at [Update an API using just the OAS definition](#update-an-api-using-just-the-oas-definition), you also defined a JSON schema that describes the response format for any request that hits the `GET /pet/{petId}` path.
+
+You can now tell the Gateway to mock response of any incoming request against the documented JSON schema. This is achieved by adding the `mockResponse` query parameter to the `PATCH` request, when updating the Tyk OAS API Definition.
+
+| Property     | Description                                               |
+|--------------|-----------------------------------------------------------|
+| Resource URL | /tyk/apis/oas/{api-id}                                    |
```
Comment (letzya) [inline tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]:
```suggestion
| Resource URL | `/tyk/apis/oas/{api-id}`                                    |
```

File: tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md [hunk 5]
```diff
@@ -566,6 +566,173 @@ Go to the `/apps` folder of your Tyk Gateway installation (by default in `/var/t
...
+| Method       | PATCH                                                     |
+| Type         | None                                                      |
+| Body         | OAS API Definition                                        |
+| Param        | Path Parameter: {api-id} Query Parameter: mockResponse    |
```
Comment (letzya) [inline tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]:
```suggestion
| Path Param |  `{api-id}`   |
|.Query Param | `mockResponse`    |
```

File: tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md [hunk 6]
```diff
@@ -566,6 +566,173 @@ Go to the `/apps` folder of your Tyk Gateway installation (by default in `/var/t
   }
 }
 ```
+
+#### Mock response from OAS definition
+
+In the OAS API Definition that you updated at [Update an API using just the OAS definition](#update-an-api-using-just-the-oas-definition), you also defined a JSON schema that describes the response format for any request that hits the `GET /pet/{petId}` path.
```
Comment (letzya) [inline tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]: Can you please update the link to use this format https://tyktech.atlassian.net/wiki/spaces/EN/pages/1888976900/Docs+contribution+guidelines+-+empower+you+to+write#6

**Review by letzya — CHANGES_REQUESTED** [review]

File: tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md [hunk 7]
```diff
@@ -566,6 +566,174 @@ Go to the `/apps` folder of your Tyk Gateway installation (by default in `/var/t
...
+| Property     | Description                                               |
+|--------------|-----------------------------------------------------------|
+| Resource URL | `/tyk/apis/oas/{api-id}`                                    |
+| Method       | PATCH                                                     |
+| Type         | None                                                      |
+| Body         | OAS API Definition                                        |
+| Path Param |  `{api-id}`   |
+|.Query Param | `mockResponse`    |
+
+```
```
Comment (letzya) [inline tyk-docs/content/getting-started/using-oas-definitions/update-api-with-oas.md]:
```suggestion
```curl
```

**Review by letzya — APPROVED** [review]

**Pull request merged**